src/main/webapp/app/localvc/vcs-repository-access-log-view/vcs-repository-access-log-view.component.ts

```diff
@@ -0,0 +1,75 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable, Subject, Subscription } from 'rxjs';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DomainService } from 'app/exercises/programming/shared/code-editor/service/code-editor-domain.service';
+import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
+import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
+import { ProgrammingExerciseParticipationService } from 'app/exercises/programming/manage/services/programming-exercise-participation.service';
+import { VcsAccessLogDTO } from 'app/entities/vcs-access-log-entry.model';
+import { HttpErrorResponse } from '@angular/common/http';
+import { AlertService } from 'app/core/util/alert.service';
+import { ArtemisSharedCommonModule } from 'app/shared/shared-common.module';
+
+@Component({
+    selector: 'jhi-vcs-repository-access-log-view',
+    templateUrl: './vcs-repository-access-log-view.component.html',
+    standalone: true,
+    imports: [ArtemisSharedCommonModule],
+})
+export class VcsRepositoryAccessLogViewComponent implements OnInit {
+    participationId: number;
+    vcsAccessLogEntries: VcsAccessLogDTO[];
+    protected dialogErrorSource = new Subject<string>();
+
+    paramSub: Subscription;
```

Comment:
Since you're subscribing & storing the subscription, can you also unsubscribe when the component is destroyed?

Reply:
Would it also work to just remove this Subscription here entirely,  as `this.route.params.subscribe` should get unsubscribed automatically at the end? It seems like I might not need the paramSub at all, or do I?

Reply:
Either one would be fine imo. I think it gets unsubscribed automatically